api.md

```diff
@@ -1822,6 +1823,33 @@ ORDER BY day;
 
 ## Utilities/Statistics [](utilities)
 
+### timescaledb_information.data_node [](#timescaledb_information-datanode)
+
+Get information on data nodes.
+
+|Name|Description|
+|---|---|
+| `node_name` | Data node name. |
+| `owner` | oid |
+| `options` | Options used when creating the data node. |
+| `node_up` | (BOOLEAN) Data node responds to ping. |
+| `num_dist_tables` | Number of distributed hypertables that use this data node. |
+| `num_dist_chunks` | Total number of chunks stored in data node. |
+| `total_dist_size` | Total amount of data stored in data node. |
+
+#### Sample Usage
+
+Get information on data nodes.
```

Comment:
This is the exact same line as line 1828, probably want to vary the wording so it doesn't look like a copy/paste.

Reply:
Did a quick variation.